lte/gateway/c/session_manager/PipelinedClient.cpp

```diff
@@ -532,4 +532,13 @@ uint32_t AsyncPipelinedClient::get_current_teid() {
   return teid;
 }
 
+void AsyncPipelinedClient::set_upf_n3_addr(std::string addr) {
+  upf_node_ip_addr = addr;
```

Comment:
What is the purpose of this function? Do we eventually intend to send a request to PipelineD?

Reply:
We intend to send to AMF. Here we are learning (receiving) N3 IP address from UPF, retaining in SMF. Sharing the IP info to AMF.

Reply:
So is `AsyncPipelinedClient` holding on to this value?

Reply:
Correct.

Reply:
Hey Venu, this is a client API. You should not store anything in this object.

Reply:
removed all the APIs, moved SessionStateEnforcer.cpp